GoogleSignIn/Tests/Unit/GIDGoogleUserTest.m

```diff
@@ -91,4 +95,48 @@ - (void)testLegacyCoding {
 }
 #endif // TARGET_OS_IOS || TARGET_OS_MACCATALYST
 
+- (void)testUpdateAuthState {
+  NSTimeInterval accessTokenExpireTime = [NSDate timeIntervalSinceReferenceDate];
+  NSTimeInterval idTokenExpireTime = [NSDate timeIntervalSinceReferenceDate] + 100;
```

Comment:
Do 100, 200, and 300 have special meaning, or are you just making sure that these times are different? I _think_ the time intervals should be different with each subsequent call to `[NSDate timeIntervalSinceReferenceDate]`. If so, maybe just drop the addition.

Reply:
@Alex-4-Git could you respond to Matt's question here.

I'd also suggest marking comment threads as resolved when you have addressed them so as not to miss those that still need to be addressed.

Reply:
In the test we use a method `XCTAssertEqualWithAccuracy`. We want to make sure the difference between the expire times is larger than the accuracy so they would not be misused. 

I have used a constant kTimeIncrement to improve the readability.